specification/dnc/resource-manager/Microsoft.DelegatedNetwork/preview/2020-08-08-preview/delegatedSubnets.json

```diff
@@ -0,0 +1,515 @@
+            "schema": {
+              "$ref": "#/definitions/DelegatedSubnets"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Resource": {
+      "description": "Represents an instance of a resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "An identifier that represents the resource."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the resource."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The type of resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "Location of the resource."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The resource tags."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "ResourceUpdateParameters": {
+      "description": "Parameters for updating a resource.",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The resource tags."
+        }
+      }
+    },
+    "DelegatedSubnets": {
+      "description": "An array of DelegatedSubnet resources.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DelegatedSubnet"
+          },
+          "description": "An array of DelegatedSubnet resources."
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "DelegatedSubnet": {
+      "description": "Represents an instance of a orchestrator.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "readOnly": true,
+          "$ref": "#/definitions/DelegatedSubnetProperties",
+          "description": "Properties of the provision operation request."
+        }
+      }
+    },
+    "DelegatedSubnetProperties": {
+      "description": "Properties of delegated subnet",
+      "type": "object",
+      "properties": {
+        "resourceGuid": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource guid."
+        },
+        "state": {
```

Comment:
long running PUTs should be using a "provisioningState" property, not "state"

Reply:
state has "provisioning" as value.

Reply:
Yes.  The async pattern described here is strict:  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creatingupdating-using-put